pkg/scheduler/framework/plugins/volumebinding/binder.go

```diff
@@ -199,6 +199,15 @@ type SchedulerVolumeBinder interface {
 	BindPodVolumes(ctx context.Context, assumedPod *v1.Pod, podVolumes *PodVolumes) error
 }
 
+// PodVolumesSeparated stores a pod's PVCs separated into bound, unbound with delayed binding (including provisioning),
+// unbound with immediate binding (including prebound) and pvs that belong to storage classes of unbound PVCs with delayed binding.
```

Comment:
please comment on each parameter below instead of having one large block of comment here.